EIPS/eip-2390.md

```diff
@@ -0,0 +1,314 @@
+---
+eip: 2390
+title: Geo-ENS
+author: James Choncholas <jchoncholas@gmail.com>
+discussions-to: https://github.com/ethereum/EIPs/pull/2390
```

Comment:
This has to be separate from the pull request itself.  You can create an issue in GitHub if you prefer.

Reply:
Issue created at https://github.com/ethereum/EIPs/issues/2959 and EIP edited in commit 41b42cd64d2f6039277754372059741180a15f22